include/alpaka/block/shared/st/Traits.hpp

```diff
@@ -152,6 +152,7 @@ namespace alpaka
                         >::type>
                     {
                         //-----------------------------------------------------------------------------
+                        ALPAKA_NO_HOST_ACC_WARNING
```

Comment:
Is this really necessary here?

Reply:
yes, and also in mem/view/Traits.hpp. Separated PRs are in the pipe, need to test other stuff before though.